README.md

```diff
@@ -71,6 +71,24 @@ Use this space to show useful examples of how a project can be used. Additional
 For more examples, please refer to the [Documentation](https://github.com/rp-rs/rp-hal)
 
 
+### Run examples
+
+Install [`uf2conv`](https://github.com/sajattack/uf2conv-rs) and [`cargo-binutils`](https://github.com/rust-embedded/cargo-binutils)
+
+```sh
+cargo install uf2conv cargo-binutils
```

Comment:
I wrote up a similar script to this yesterday, was considering adding it directly into the repo.
You've missed a step here, which is to run
rustup component add llvm-tools-preview

Reply:
Ah yes will add that